src/averageAges.js

```diff
@@ -7,19 +7,28 @@
  * function calculates average age only for men who died in this century
  *
  * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
+ * Divide year of personson's death by 100: Math.ceil(personson.died / 100)
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
+
+function averageAge(persons) {
+  return persons
+    .map(person => person.died - person.born)
+    .reduce((firstPerson, secondPerson) => (
```

Comment:
- you may try destructuring here, in that case you won't need to use .map method
- first arg of reduce callback is not actually its first element, but the accumulator
```suggestion
    .reduce((acc, { died, born }) => (
```